packages/transport-redis/src/RedisTransport.js

```diff
@@ -60,7 +80,8 @@ class RedisTransport {
   }
 
   enqueue(queueName, obj) {
-    return this._getEnqueuer(queueName).add(obj, { removeOnComplete: true, timeout: JOB_TIMEOUT });
+    const opts = assign({ jobId: get(obj, 'headers.id') }, BULL_OPTIONS);
```

Comment:
I'm pretty sure that jobId changed to id in bull v3

Reply:
https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queueadd